master/pkg/model/task_container_defaults.go

```diff
@@ -51,16 +51,22 @@ type TaskContainerDefaultsConfig struct {
 
 	LogPolicies expconf.LogPoliciesConfig `json:"log_policies"`
 
+	PreemptionTimeout int `json:"preemption_timeout,omitempty"`
+
 	// TODO(DET-9856) we should probably eventually move this to expconf and allow setting
 	// on a per task level.
 	Kubernetes *KubernetesTaskContainerDefaults `json:"kubernetes"`
 }
 
+// DefaultPreemptionTimeout is the number of seconds to wait for preempted task to exit gracefully.
+const DefaultPreemptionTimeout = 60 * 60
```

Comment:
(not a blocker) For my understanding, why do we have two default preemption timeout values? One is 60 * 60 and the other in `schemas/expconf/v0/experiment.json` is null.

Reply:
the experiment config in `schemas/expconf/v0/experiment.json` will get overridden by the task container defaults if it's not set. we can't set defaults in expconf because we won't know if it's unset or not